com.revolutionarygamesstudio.ThriveLauncher.appdata.xml

```diff
@@ -0,0 +1,174 @@
+		</p>
+		
+		<p>Current key features:</p>
+		<ul>
+			<li>
+			Control an individual member of your species and survive the environment
+			</li>
+			<li>
+			Predate on other species, use photosynsthesis or scavenge for resources
+			</li>
+			<li>
+			Edit your species to make it more successful
+			</li>
+			<li>
+			Compete with other species emerging on your planet via an evolution simulation
+			</li>
+			<li>
+			Explore different biomes
+			</li>
+			<li>
+			Fight other cells with multiple cellular level weapons
+			</li>
+			<li>
+			Try different gameplay styles by specializing in different energy sources in subsequent playthroughs
+			</li>
+			<li>
+			Learn about biology by using real compounds, organelles or parts inspired by real science
+			</li>
+			<li>
+			Spread your species via the biome map
+			</li>
+			<li>
+			Review and plan future actions by looking at population simulation results and graphs
+			</li>
+			<li>
+			Learn the basics of the game with a light interactive tutorial
+			</li>
+		</ul>
+		
+		<p>
+		The major goals of Thrive are to create engaging, compelling gameplay that respects our players’ intelligence, and remain as accurate as possible in our depiction of known scientific theory without compromising the former. Thrive is an open-source project, and anyone with game development skill is welcome to join our team. The game uses the open-source Godot engine with the C# programming language.
+		</p>
+		
+		<p>
+		If you don't have game development skills, you are still welcome to join our fan community. We would love to have you along for the long ride! 
+		</p>
+	</description>
+
+	<categories>
+		<category>Game</category>
+	</categories>
+
+	<keywords>
+		<keyword>Spore</keyword>
+	</keywords>
+
+	<url type="homepage">https://revolutionarygamesstudio.com/</url>
+	<url type="bugtracker">https://github.com/Revolutionary-Games/Thrive/issues</url>
+	<url type="faq">https://revolutionarygamesstudio.com/faq/</url>
+	<url type="help">https://wiki.revolutionarygamesstudio.com/wiki/Main_Page</url>
+	<url type="donation">https://www.patreon.com/thrivegame</url>
+	<url type="translate">https://github.com/Revolutionary-Games/Thrive/blob/master/doc/working_with_translations.md</url>
+	<!--<url type="vcs-browser">https://github.com/Revolutionary-Games/Thrive</url>-->
+	<!--<url type="contribute">https://github.com/Revolutionary-Games/Thrive#getting-involved</url>-->
```

Comment:
They caused build errors

Reply:
Don't worry about that. Flathub has a newer version of appstream-glib (the lib that parses that file), so it will pass even if it doesn't work on your local machine.

Reply:
I rather not, I run Fedora and if it doesn't run on Fedora I won't uncomment it.

Reply:
You can use `org.flatpak.Builder` from Flathub instead.